LIFECYCLE.md

```diff
@@ -128,24 +159,12 @@ echo false > $MODULE_ENABLED_RESULT
 
 Module `some-module` is explicitly disabled in `modules/values.yaml` but enabled by `someModuleEnabled` key in ConfigMap/addon-operator. Thus enabled script is executed and returns `false`. So the final result is that the module is disabled.
 
-# Tasks queue
+# Task queues
 
-Addon-operator cycle works like a simple FIFO queue. The Addon-operator processes an event, creates a task and adds it to the queue. The queue handler runs the current task and proceeds to the next. Each task is processed until successful completion. In case of an error, the task is returned to the top of the queue and executed after a 5 seconds delay. When executing tasks for the `onKubernetesEvent` and `schedule` events, the queue handler may ignore the execution errors if the `allowFailure: true` flag is specified in the binding configuration.
+Task queues are a simple FIFO queues. The Addon-operator process an event, creates a task and adds it to the particular named queue. Each named queue has its own queue handler which runs the top task and proceeds to the next.
+
+Each task is processed until successful completion. In case of an error, the task is returned to the top of the queue and executed after a 5 seconds delay. When executing tasks for the `kubernetes` and `schedule` events, the queue handler ignores the execution errors if the `allowFailure: true` flag is specified in the binding configuration.
```

Comment:
```suggestion
Each task is processed until successful completion. In case of an error, the task is returned to the start of the queue and executed after a 5 seconds delay. When executing tasks for the `kubernetes` and `schedule` events, the queue handler ignores the execution errors if the `allowFailure: true` flag is specified in the binding configuration.
```